src/Profile/RedisVersion320.php

```diff
@@ -276,6 +276,11 @@ public function getSupportedCommands()
             'GEODIST' => 'Predis\Command\GeospatialGeoDist',
             'GEORADIUS' => 'Predis\Command\GeospatialGeoRadius',
             'GEORADIUSBYMEMBER' => 'Predis\Command\GeospatialGeoRadiusByMember',
+
+            /* ---------------- Redis 5.0 ---------------- */
+
+            /* commands operating on sorted sets */
+            'ZPOPMAX' => 'Predis\Command\ZSetRemoveMaxByScore',
```

Comment:
probably this should be a different profile, no? Like `RedisVersion500.php`?

Reply:
I think you are right.
I will update that :).

I thought no one gonna look into that anymore 😃.

Thank you.

Reply:
Added the new profile.
@hannesvdvreken & @tillkruss re-check :)